### PR TITLE
Rate activity

### PR DESCRIPTION
**Pull request by procrasprincess**

### CODE REVIEW

**Comment by ChexWarrior** [comment]
Relating this to #61

File: src/DataFixtures/ActivityFixture.php [hunk 1]
```diff
@@ -33,15 +33,18 @@ public function load(ObjectManager $manager)
...
             $activity->setZip((int)$faker->postcode);
             $activity->setCity($faker->city);
             $activity->setState($faker->state);
             $activity->setCountry($faker->country);
             $activity->setLatitude((float)$faker->latitude);
             $activity->setLongitude((float)$faker->longitude);
 
+            $activity->setThumbsUp(i);
+            $activity->setThumbsDown(i);
+
```
Comment (ChexWarrior) [inline src/DataFixtures/ActivityFixture.php]: @procrasprincess Don't forget the $ in front of PH variables here!

**Comment by ChexWarrior** [comment]
@procrasprincess This looks good, have you run a test yet though? You'll also need to generate a migration to actually make changes to the database.  See [here](https://symfony.com/doc/current/doctrine.html#migrations-adding-more-fields) for details.

**Comment by procrasprincess** [comment]
> @procrasprincess This looks good, have you run a test yet though? You'll also need to generate a migration to actually make changes to the database. See [here](https://symfony.com/doc/current/doctrine.html#migrations-adding-more-fields) for details.

**Comment by procrasprincess** [comment]
I haven’t figured out the database part yet. Just wanted to make sure I got the rest right first

**Comment by ChexWarrior** [comment]
Sounds good, we can go over it together at the next meeting if you'd like, but I encourage you to try on your own too.

**Pull request closed without merging**